pkg/masking/config_mask_test.go

```diff
@@ -98,3 +98,13 @@ func TestConfig_WhenRemoveSecretsFromDremioConf(t *testing.T) {
 		t.Errorf("trustStorePassword was not masked")
 	}
 }
+
+func TestPATMask(t *testing.T) {
```

Comment:
see notes above..good test but we need test for -t as well (or remove -t)

Reply:
I added a test for `-t` I also improved the regex match.

I run two cibuilds and they both passed ok